include/CBawFile.h

```diff
@@ -7,27 +7,40 @@
 #define __IRR_BAW_FILE_H_INCLUDED__
 
 #include "stdint.h"
-#include "IMeshBuffer.h"
+#include "IMesh.h"
 
-namespace irr { namespace core
+namespace irr { 
+
+namespace scene 
+{ 
+	class ICPUSkinnedMesh;
+	class SCPUSkinMeshBuffer;
+	class CFinalBoneHierarchy;
+}
+
+namespace core
 {
 
 #include "irrpack.h"
 	//! Cast pointer to block of blob-headers to BlobHeader* and easily iterate and/or access members
-	struct BlobHeader
+	struct BlobHeaderV1
 	{
 		uint32_t blobSize;
 		uint32_t blobSizeDecompr;
 
-		uint64_t compressionType : 3;
-		uint64_t blobType : 7;
+		uint8_t compressionType;
+		uint8_t blobType;
```

Comment:
I'm going to be a dick and change this again, could we have:
`uint8_t compressionType;
uint8_t dummy[3];
uint32_t blobType; //go to the wiki page for BlobType Registry`

Order matters, I want to keep everything 64bit and type-aligned